docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -134,6 +136,8 @@ def topstories(context: AssetExecutionContext) -> MaterializeResult:
     )
 ```
 
+Dagster provides a \[standard set of metadata keys]\(#standard-asset metadata-entry-reference) that can be used for common types of metadata, such as an asset's URI or column schema. Note: These entries are intended to be a starting point, and we encourage you to create your own metadata keys that make sense within the context of your data platform.
```

Comment:
```suggestion
Dagster provides a [standard set of metadata keys](#standard-asset metadata-entry-reference) that can be used for common types of metadata, such as an asset's URI or column schema. **Note**: These entries are intended to be a starting point, and we encourage you to create your own metadata keys that make sense within the context of your data platform.
```

Reply:
For some reason these slashes keep getting automatically corrected. Any idea why?

Reply:
oo I think I know why - fixed

Reply:
I don't, but I'll look into it when I add the table update